index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+
+    <!-- Font awesome link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+    <title>COP27</title>
+  </head>
+  <body>
+    <header>
+
+      <!-- Contact links from font awesome -->
+      <div id="secondary-nav-bar" class="container">
+        <div class="wrapper">
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+
+      <!-- Navbar with sub-titles link -->
+      <div id="nav-bar">
+        <a href="#" id="logo">COP27<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
+          <a href="#" class="home-link">Home</a>
+          <a href="#program">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsors">Sponsors</a>
+          <a href="./about.html">About</a>
+        </nav>
+        <a href="#" id="menu-toggler">
+
+          <!-- Scalable vector graphics for hamburger -->
+          <svg
+            class="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </a>
+      </div>
+    </header>
+
+    <!-- Title welcome page section -->
+    <section id="section1" class="container">
+      <div class="wrapper">
+        <img src="img/bg-bob.png" class="shape-set" alt="" />
+        <h1>World Climate Summit - COP27</h1>
+        <h4>The Investment COP</h4>
+        <h2>"Sustainable Environment"</h2>
+        <p class="description">
+          World Climate Summit -
+          The Investment COP will utilise a decade of experience to bring together
```

Comment:
- [ ] please ensure that your partners are showing beside the footer of the about page
- [ ] also, please follow the original design and make the footer  of the about page mobile version `white` color
yours
![Screenshot (548)](https://user-images.githubusercontent.com/65298048/197073972-371d6013-f5bf-4c4c-86d5-e24ce0118eb3.png)
behance
![Screenshot (547)](https://user-images.githubusercontent.com/65298048/197074021-a1c9ead5-504d-40c5-bb99-e26bb3739252.png)

Reply:
Great job implementing the required changes :smile: :+1: